viewer/src/datamodels/cad/CadMaterialManager.ts

```diff
@@ -90,12 +94,19 @@ export class CadMaterialManager {
     const nodeTransformTextureBuilder = new NodeTransformTextureBuilder(maxTreeIndex + 1, nodeTransformProvider);
     nodeTransformTextureBuilder.build();
 
-    nodeAppearanceProvider.on('changed', () => {
-      this.updateMaterials(modelIdentifier);
-    });
-    nodeTransformProvider.on('changed', () => {
-      this.updateMaterials(modelIdentifier);
-    });
+    const materialUpdateFrequency = 150;
```

Comment:
that how it looks like (debounce in reveal updates is turned off)

https://user-images.githubusercontent.com/4989157/121914315-70d50680-cd32-11eb-9882-b61289b9a674.mov

Reply:
Thx, I agree that the current delay was too long. I have replaced debounce with throttle to ensure some rebuilds will be triggered when doing frequent update. 

I agree that it might be best to have this on the app side, but as we have taken ownership of the different filtering mechanisms we need to control this on our end somehow. When the user e.g. creates a `ByNodePropertyNodeSet` and executes a filter he/she's not in control of how do throttling, so I think we need to handle this and find the best values.

Reply:
> he/she's not in control of how do throttling

ah, I see, thanks for the explanation